apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -39,26 +39,50 @@ $(function() {
 
 {{#submission}}
     {{^parse-error}}
-    <div class="pl-order-blocks-answer-container">
-        <div class="card-header">
-            Your answer:
+    <div class="submission_container">
+        <ul class="list-group submission_header">
+            <li class="list-group-item"><strong>Your answer: </strong>
```

Comment:
You shouldn't be reverting the order blocks CSS to the old version, see above comment.

Reply:
done